lib/chat_api/messages/notification.ex

```diff
@@ -58,13 +58,7 @@ defmodule ChatApi.Messages.Notification do
     Logger.info("Sending message notification: :sms")
 
     Task.start(fn ->
-      # TODO: implement me!
-      #
-      # First, check if message parent conversation has `source: "sms"`
-      # Also check that the conversation customer has a valid phone number
-      # If true, use the message account ID to check for a twilio_authorization record
-      # If one exists, use it to send a message to the customer's phone number
-      nil
+      ChatApi.Twilio.Notification.notify_sms(message)
```

Comment:
Should we log an error when `notify_sms/1` returns one?

Reply:
Yeah good idea. Will push it up tn